atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -114,6 +121,9 @@ def __init__(self, tasker, workflow,
         self.to_koji_submitter = to_koji_submitter
         self.to_koji_pkgowner = to_koji_pkgowner
         self.submitter = self.DEFAULT_SUBMITTER
+        self.osbs = None
```

Comment:
Is `osbs` used outside `_fetch_full_logs()`?

Reply:
No, I think I was being overly cautious here. Fixed.